library/general/src/modules/Report.rb

```diff
@@ -422,6 +423,44 @@ def ErrorAnyQuestion(headline, message, yes_button_message, no_button_message, f
       ret
     end
 
+    # Question presented via the Yast2::Popup class
+    #
+    # It works like any other method used to present yesno_messages, but it
+    # delegates drawing the pop-up to {Yast2::Popup.show}, in case the message
+    # must be presented to the user (which can be configured via
+    # {#DisplayYesNoMessages}).
+    #
+    # All the arguments are forwarded to #{Yast2::Popup.show} almost as-is, but
+    # some aspects must be observed:
+    #
+    #   - The argument :timeout will be ignored, the timeout fixed in the Report
+    #   module will always be used instead (again, see {#DisplayYesNoMessages}).
+    #   - The button ids must be :yes and :no, to honor the Report API.
```

Comment:
to be honest, I expect that here we should use by default `buttons: :yes_no` so if you omit this param, you still get yes_no popup. Default `:ok` looks wrong for me. So in other words, when you call `if Report.yesno_popup("text")` it will always return false, as it return `:ok` and not `:yes`

So here should be only note that if buttons was overwritten, only `:yes` symbol is considered for bootlean return value.

Reply:
Done. `:yes_no` is now used by default.

About `:yes` being considered, that's already documented in the `@return`.